### PR TITLE
first pass at werewolf hunger

### PR DESCRIPTION
**Pull request by tomcraven**

### CODE REVIEW

File: nodes/villager/Villager.gd [hunk 1]
```diff
@@ -283,6 +286,9 @@ func hurt():
 	$AudioStreamPlayer2D.stream = load("res://assets/sounds/stab.ogg")
 	$AudioStreamPlayer2D.play()
 
+	moon.amend_crescent(0.25)
```
Comment (marc-jones) [inline nodes/villager/Villager.gd]: Maybe make the 0.25 a class variable?

File: scripts/Moon.gd [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+
+var face = 1.0
+
+var hunger_duration_seconds = 30.0
+
+func _ready():
+	set_crescent(1.0)
+
+func _process(delta):
+	amend_crescent((1.0 / hunger_duration_seconds) * -delta)
```
Comment (marc-jones) [inline scripts/Moon.gd]: We need negative delta handling :stuck_out_tongue:

**Review by marc-jones — CHANGES_REQUESTED** [review]
Couple of comments, but looks good.

File: scripts/Moon.gd (hunk 2 above)
Comment (tomcraven) [reply scripts/Moon.gd]: b8226a4

File: nodes/villager/Villager.gd (hunk 1 above)
Comment (tomcraven) [reply nodes/villager/Villager.gd]: 53f9b83

**Pull request merged**